### PR TITLE
Power v10

### PR DESCRIPTION
**Pull request by tyshko5**
**Summary of changes**
Changes introduced in this pull request:
- 

**Reference issue to close (if applicable)**

Closes 

**Other information and links**

### CODE REVIEW

File: power_v10/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+keywords = ["filecoin", "web3", "wasm"]
+
+[dependencies]
+fil_actors_runtime_v10 = { workspace = true }
+frc42_dispatch = "3.0.0"
+fvm_shared = { version = "2.0.0-alpha.2", default-features = false }
+fvm_ipld_hamt         = { workspace = true }
+num-derive            = { workspace = true }
+num-traits            = { workspace = true }
+indexmap = { version = "1.8.0", features = ["serde-1"] }
```
Comment (creativcoder) [inline power_v10/Cargo.toml:22]: workspace = true

File: power_v10/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "Filecoin Core Devs",
+]
+edition = "2018"
+repository = "https://github.com/ChainSafe/forest"
+keywords = ["filecoin", "web3", "wasm"]
+
+[dependencies]
+fil_actors_runtime_v10 = { workspace = true }
+frc42_dispatch = "3.0.0"
+fvm_shared = { version = "2.0.0-alpha.2", default-features = false }
```
Comment (creativcoder) [inline power_v10/Cargo.toml:18]: workspace = true

File: power_v10/src/lib.rs [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    // * Deprecated in v2
+    // OnConsensusFault = 7,
+    SubmitPoRepForBulkVerify = 8,
+    CurrentTotalPower = 9,
+    // Method numbers derived from FRC-0042 standards
+    CreateMinerExported = frc42_dispatch::method_hash!("CreateMiner"),
+    NetworkRawPowerExported = frc42_dispatch::method_hash!("NetworkRawPower"),
+    MinerRawPowerExported = frc42_dispatch::method_hash!("MinerRawPower"),
+    MinerCountExported = frc42_dispatch::method_hash!("MinerCount"),
+    MinerConsensusCountExported = frc42_dispatch::method_hash!("MinerConsensusCount"),
```
Comment (creativcoder) [inline power_v10/src/lib.rs:47]: the (current) builtin-actors repo do not have the above 5 variants these, curious as to why we've added it here?

Comment (creativcoder) [reply power_v10/src/lib.rs:47]: those are there, please ignore

**Review by creativcoder — APPROVED** [review]

File: power_v10/Cargo.toml (hunk 2 above)
Comment (tyshko5) [reply power_v10/Cargo.toml]: Fixed

File: power_v10/Cargo.toml (hunk 1 above)
Comment (tyshko5) [reply power_v10/Cargo.toml]: Fixed

**Pull request merged**